crates/storage_impl/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "storage_impl"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+# First Party dependencies
+masking = { version = "0.1.0", path = "../masking" }
+
+# Third party crates
+bb8 = "0.8.1"
+diesel = { version = "2.1.0", default-features = false, features = ["postgres"] }
+async-bb8-diesel = { git = "https://github.com/oxidecomputer/async-bb8-diesel", rev = "be3d9bce50051d8c0e0c06078e8066cc27db3001" }
```

Comment:
Can we remove `async_bb8_diesel` and `bb8` dependencies from the `router` crate, or do we need them?

Reply:
I assume they are still needed since 
```
impl StorageInterface for Store {}
```

is done in router crate... once we split this out we won't need them